docs/user/metrics/labeled_booleans.md

```diff
@@ -0,0 +1,134 @@
+
+{{#include ../../tab_header.md}}
+
+<div data-lang="Kotlin" class="tab">
+
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Accessibility
+Accessibility.features["screen_reader"].set(isScreenReaderEnabled())
+Accessibility.features["high_contrast"].set(isHighContrastEnabled())
+```
+
+There are test APIs available too:
+
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Accessibility
+// Was anything recorded?
+assertTrue(Accessibility.features["screen_reader"].testHasValue())
+assertTrue(Accessibility.features["high_contrast"].testHasValue())
+// Do the booleans have the expected values?
+assertEquals(True, Accessibility.features["screen_reader"].testGetValue())
+assertEquals(False, Accessibility.features["high_contrast"].testGetValue())
+// Did we record any invalid labels?
+assertEquals(1, Accessibility.features.testGetNumRecordedErrors(ErrorType.InvalidLabel))
+```
+
+</div>
+
+<div data-lang="Swift" class="tab">
+
+```Swift
+Accessibility.features["screen_reader"].set(isScreenReaderEnabled())
+Accessibility.features["high_contrast"].set(isHighContrastEnabled())
+```
+
+There are test APIs available too:
+
+```Swift
+@testable import Glean
+
+// Was anything recorded?
+XCTAssert(Accessibility.features["screen_reader"].testHasValue())
+XCTAssert(Accessibility.features["high_contrast"].testHasValue())
+// Do the booleans have the expected values?
+XCTAssertEqual(true, try Accessibility.features["screen_reader"].testGetValue())
+XCTAssertEqual(false, try Accessibility.features["high_contrast"].testGetValue())
+```
+
+</div>
+
+<div data-lang="Python" class="tab">
+
+```Python
+from glean import load_metrics
+metrics = load_metrics("metrics.yaml")
+
+metrics.accessibility.features["screen_reader"].set(
+    is_screen_reader_enabled()
+)
+metrics.accessibility.features["high_contrast"].set(
+    is_high_contrast_enabled()
+)
+```
+
+There are test APIs available too:
+
+```Python
+# Was anything recorded?
+assert metrics.accessibility.features["screen_reader"].test_has_value()
+assert metrics.accessibility.features["high_contrast"].test_has_value()
+# Do the booleans have the expected values?
+assert metrics.accessibility.features["screen_reader"].test_get_value()
+assert not metrics.accessibility.features["high_contrast"].test_get_value()
+# Did we record any invalid labels?
+assert 1 == metrics.accessibility.features.test_get_num_recorded_errors(
+    ErrorType.INVALID_LABEL
+)
+```
+
+</div>
+
+{{#include ../../tab_footer.md}}
+
+## Limits
+
+* Labels must conform to the [label formatting regular expression](index.md#label-format).
+
+* Labels support lowercase alphanumeric characters; they additionally allow for dots (`.`), underscores (`_`) and/or hyphens (`-`).
+
+* Each label must have a maximum of 60 bytes, when encoded as UTF-8.
+
+* If the labels are specified in the `metrics.yaml`, using a different label will be replaced with the special value `__other__`.
```

Comment:
I stumbled upon the "different".
Maybe "unknown" or "not listed" or something to make it clear that it's "different from the listed ones"